### PR TITLE
Create a document to  setup database for scalar DB/scalar DL deployment in EKS.

### PR DESCRIPTION
**Pull request by inv-jishnu**
Created a new setup database document for scalar DB/scalar DL deployments in EKS with all supported databases.
The section for `DynamoDB` is taken from existing [SetupDatabase](https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/SetupDatabase.md#dynamodb) document.

### CODE REVIEW

**Review by scalar-boney — COMMENTED** [review]
Left some comments.
Please take a look when you get a chance?

File: docs/SetupDatabaseForEKS.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+* AWS RDS for Oracle
+* AWS RDS for SQL Server
+
+In this section, you will create an AWS RDS for MySQL/PostgreSQL/Oracle/SQL Server.
+
+### Recommendations
+
+* You should select either [General Purpose SSD storage](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage.GeneralSSD) or [Provisioned IOPS SSD storage](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS) for storage according to your requirements.
+    * You should select the storage type and amount of storage to ensure that adequate IOPS is available as per your requirements.
+* You should check the [official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_BestPractices.html#CHAP_BestPractices.DiskPerformance) to learn more about AWS RDS best practices.
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:56]: Could you please standardize the use of `official guide`, `official documentation`, and  `AWS official guide`?

It's ok if you added purposefully.

File: docs/SetupDatabaseForEKS.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+By default, monitoring is enabled on DynamoDB.
+
+More information can be found in [DynamoDB Metrics and Dimensions documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html).
+
+## AWS RDS for MySQL, PostgreSQL, Oracle, SQL Server
+
+Amazon Relational Database Service(RDS) is a cloud-based distributed relational database service provided by Amazon Web Services.
+It provides MySQL, PostgreSQL, Oracle, SQL Server databases.
+Scalar DL and Scalar DB support the following AWS RDS databases.
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:44]: Ditto

File: docs/SetupDatabaseForEKS.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+### Recommendations
+
+* You should select either [General Purpose SSD storage](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#Concepts.Storage.GeneralSSD) or [Provisioned IOPS SSD storage](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS) for storage according to your requirements.
+    * You should select the storage type and amount of storage to ensure that adequate IOPS is available as per your requirements.
+* You should check the [official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_BestPractices.html#CHAP_BestPractices.DiskPerformance) to learn more about AWS RDS best practices.
+
+### Steps
+
+You can follow this [official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create a database instance.
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:60]: I think we should reduce the wide usage of `you can`.

File: docs/SetupDatabaseForEKS.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+### Monitor DynamoDB
+
+By default, monitoring is enabled on DynamoDB.
+
+More information can be found in [DynamoDB Metrics and Dimensions documentation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html).
+
+## AWS RDS for MySQL, PostgreSQL, Oracle, SQL Server
+
+Amazon Relational Database Service(RDS) is a cloud-based distributed relational database service provided by Amazon Web Services.
+It provides MySQL, PostgreSQL, Oracle, SQL Server databases.
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:43]: I think it is redundant.

File: docs/SetupDatabaseForEKS.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+### Steps
+
+You can follow this [official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create a database instance.
+
+### Scale Performance
+
+You can scale up or scale down the instance and memory used. In addition, AWS also has the option to provide additional storage.
+You can also add read replicas to improve read-heavy database workloads.
+
+#### Autoscaling
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:67]: I think, need to reduce the use of `you can`.

File: docs/SetupDatabaseForEKS.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+### Scale Performance
+
+There are various options to manage performance and scaling for Aurora DB clusters and DB instances.
+You can read more about this [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html).
+
+#### Autoscaling
+
+Autoscaling can be enabled in Amazon Aurora if you have created the Amazon Aurora cluster with one primary instance and at least on Aurora Replica. You can read more about it [here](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Integrating.AutoScaling.html).
+
+### Monitor Amazon Aurora
```
Comment (scalar-boney) [inline docs/SetupDatabaseForEKS.md:98]: Is it possible to standardize the monitoring section?

**Review by scalar-boney — COMMENTED** [review]
Sorry, Left some suggestions.

**Comment by inv-jishnu** [comment]
Closing this PR.
I will create a new PR by adding one supported database at a time for making reviews easy.

**Pull request closed without merging**